src/main.js

```diff
@@ -25,18 +34,49 @@ randomCoverButton.addEventListener("click", randomCover);
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
-function randomCover() {
+function makeRandomCover() {
   var randomImage = covers[getRandomIndex(covers)];
   var randomTitle = titles[getRandomIndex(titles)];
   var randomTaglineOne = descriptors[getRandomIndex(descriptors)];
   var randomTaglineTwo = descriptors[getRandomIndex(descriptors)];
 
-  var currentCover = new Cover(randomImage, randomTitle, randomTaglineOne, randomTaglineTwo);
+  currentCover = new Cover(randomImage, randomTitle, randomTaglineOne, randomTaglineTwo);
 
   coverTitle.innerText = currentCover.title;
   coverImage.src = currentCover.cover;
   coverTagline1.innerText = currentCover.tagline1;
   coverTagline2.innerText = currentCover.tagline2;
+}
+
+function hideVariable(variable){
+  variable.classList.add("hidden")
+
+}
+
+function showVariable(variable){
+  variable.classList.remove("hidden")
+}
+
+function showForm(){
+  hideVariable(randomCoverButton);
+  hideVariable(saveCoverButton);
+  hideVariable(mainPage);
+  showVariable(homeButton);
+  showVariable(formPage);
+}
+
+function loadHome() {
+  showVariable(randomCoverButton);
+  showVariable(saveCoverButton);
+  showVariable(mainPage);
+  hideVariable(homeButton);
+  hideVariable(formPage);
+}
 
-  console.log(currentCover);
+function showSavedCovers() {
+  hideVariable(mainPage);
+  showVariable(savedCoversPage);
+  hideVariable(randomCoverButton);
+  hideVariable(saveCoverButton);
+  showVariable(homeButton);
```

Comment:
These are clean and readable but you are writing out the function a bunch of times. I think it is good the way it is but I wonder if you could combine the hide and show functions some how. maybe the function hides everything first and then pass the elements you want to show as an argument? Keep an eye on these as the project grows and you might see a pattern